### PR TITLE
docs: update man page for H.264 stuff

### PR DESCRIPTION
**Pull request by metalefty**
- Update man page for xrdp.ini for newly added frame capture interval
- Add man page for new config file gfx.toml

To be backported to v0.10.

### CODE REVIEW

File: docs/man/gfx.toml.5.in [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+.SH "CODEC"
+\fB[codec]\fR section defines preferred codec order and encoder. The options
+to be specified in this section are the following:
+
+.TP
+\fBorder\fR = \fI<array>\fR
+Define the order in which codecs should be used, "H.264" and "RFX".
+Specify as an array of strings like \fB[ "H.264", "RFX ]\fR.
+Strings in the array are case-insensitive, while others are case-sensitive.
```
Comment (matt335672) [inline docs/man/gfx.toml.5.in:35]: Which others? Might be better to just say "Strings in the array are case-insensitive."

File: docs/man/gfx.toml.5.in [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+.TP
+\fBh264_encoder\fR = \fI<string>\fR
+Specify a preferred H.264 encoder, \fB"x264"\fR or \fB"OpenH264"\fR.
+This parameter takes effect only when more than one encoder is
+enabled at compile time. If only one encoder is enabled, the encoder
```
Comment (matt335672) [inline docs/man/gfx.toml.5.in]: Replace "If only one encoder is enabled" with "If only one H.264 encoder is enabled."

File: docs/man/gfx.toml.5.in [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+will be used regardless the value of this parameter. Defaults to \fB"x264"\fR
+if not specified or if an invlaid encoder is specified.  The encoder name is
```
Comment (matt335672) [inline docs/man/gfx.toml.5.in]: invlaid -> invalid.

File: docs/man/gfx.toml.5.in [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+case-insensitive, while others are case-sensitive.
```
Comment (matt335672) [inline docs/man/gfx.toml.5.in]: Same comment about "while others are case-sensitive." Do we need it?

File: docs/man/gfx.toml.5.in [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+Sets an upper limit of the bitrate in bps.
+.TP
+\fBMaxFrameRate\fR = \fI<float>\fR
+Sets the maximum frame rate that the encoder will process per second.
+
+.SH "CONNECTION TYPES"
+
+\fB[x264]\fR and \fB[OpenH264]\fR section are tables (also known as
+dictionaries) that have subtables with connection types in their keys.
+For exapmle, \fB[x264.lan]\fR, \fB[OpenH264.wan]\fR.
```
Comment (matt335672) [inline docs/man/gfx.toml.5.in]: exapmle -> example

File: docs/man/gfx.toml.5.in [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+
+You can configure different parameters such as bitrate for the encoder
+per connection type. Define the default parameter set first, this will be
+inherited to all connection types unless explicitly overridden in each
+connection type.
+
+List of available connection types are:
+
+.B lan, wan. broadband_high, satellite, broadband_low, modem, and default
```
Comment (matt335672) [inline docs/man/gfx.toml.5.in]: The comma after wan appears to be a stop.

**Review by matt335672 — COMMENTED** [review]
That's a great job!

Very minor comments attached.

File: docs/man/gfx.toml.5.in (hunk 1 above)
Comment (metalefty) [reply docs/man/gfx.toml.5.in:35]: Makes sense.

**Pull request merged**